autofit/graphical/messages/abstract.py

```diff
@@ -403,10 +424,10 @@ def _get_mean_variance(mean: np.ndarray, covariance: float
         return mean, variance
 
     def __call__(
-        self, 
-        x: np.ndarray, 
-        _variables: Optional[Tuple[str]]=('x')
-    ) -> Union[np.ndarray, Tuple[np.ndarray, np.ndarray]]:
+        self,
+        x: np.ndarray,
+        _variables: Optional[Tuple[str]] = ('x',)
```

Comment:
Given that _variables has a default value I'd be inclined to assume it's not Optional?

Reply:
passing `None` means that no gradient is returned, passing `()` means that the gradient returned is empty.